src/armscan_env/envs/labelmaps_navigation.py

```diff
@@ -272,32 +269,22 @@ def apply_volume_transformation(
         volume: sitk.Image,
         optimal_action: ManipulatorAction,
     ) -> (sitk.Image, ManipulatorAction):  # type: ignore
-        small_random_z_rotation = np.random.uniform(-20, 20)
-        small_random_x_rotation = np.random.uniform(-5, 5)
-        small_random_x_translation = np.random.uniform(-25, 25)
-        small_random_y_translation = np.random.uniform(-5, 5)
-        transformed_optimal_action = ManipulatorAction(
-            rotation=(
-                optimal_action.rotation[0] + small_random_z_rotation,
-                optimal_action.rotation[1] + small_random_x_rotation,
-            ),
-            translation=(
-                optimal_action.translation[0],
-                optimal_action.translation[1] + small_random_y_translation,
-            ),
+        volume_transformation = ManipulatorAction(
+            rotation=(np.random.uniform(-20, 20), np.random.uniform(-5, 5)),
```

Comment:
the transformation and the sampling should be separated. This function should receive a sampled rotation and translation instead of sampling here

Reply:
where do you see the action sampling function, in the env or in the ManipulatorAction class? Should the range be configurable?

Reply:
The rotation and translation are sampled here. This is not good design (violated open closed and single responsibility principle). Instead, the function should take a rotation and translation as args and sampling should happen in downstream code